### DISCUSSION TITLE
unitycontext.send data limit?

### DISCUSSION THREAD

**Discussion by BrandonNielsen** [jeffreylanters/react-unity-webgl#198] [Question and Answer]
I am working on a project that is sending large amounts of JSON data to unity through react. It works when I send most of my data, but the larger portions crash my page. Is there a certain data limit that can be sent through the send method?

**Comment by jeffreylanters** [comment]
I've personally never ran into a limit, about what amount of data are we talking, like hundreds of kilobytes or more like megabytes or even larger amounts?

**Reply by BrandonNielsen** [reply]
First off, thank you for your quick response!

The largest file that I have successfully sent through is about 8 megabytes. The first file that does not send is about 12. If there is a limit then it would probably be around that size. I am currently looking into splitting up the JSON data, sending it multiple times and refactoring it on unity's side. Thank you for the help.

**Reply by jeffreylanters** [reply]
No problem at all! I've been looking into your question, but I was unable to get any clear answer. I hope it does not have to do with the amount of free memory on your browser, since this would make it a device specific issue. Unity removed manual memory size allocation, it should balance dynamically. When looking at your browsers memory monitoring, are you seeing any spikes while trying to send large amounts of data?
